cmake/CeleritasConfig.cmake.in

```diff
@@ -43,10 +43,21 @@ endif()
 include(CMakeFindDependencyMacro)
 
 if(CELERITAS_USE_CUDA)
-  enable_language(CUDA)
   find_dependency(CUDAToolkit REQUIRED QUIET)
+  if(CELERITAS_USE_VecGeom AND NOT CMAKE_CUDA_COMPILER_LOADED)
+    include(CheckLanguage)
+    check_language(CUDA)
+    if(CMAKE_CUDA_COMPILER)
```

Comment:
If the user didn't include an enable_language or set the language in project because they weren't expecting to build CUDA code, then the language could be loaded but isn't.

It is my understanding that it's only needed for CUDA due to RDC but @pcanal should confirm on that